src/PPN-microbench/cpu_frequency.cpp

```diff
@@ -47,9 +53,9 @@ void CPUFrequency::run() {
 
     for (int coresToExecute = 1; coresToExecute <= nbTestingCores; coresToExecute++) { // Main for, equivalent to a graph
         for (int coresExecuted = 1; coresExecuted <= coresToExecute; coresExecuted++) { // For every core count, equivalent to a point in a graph
-            for (int sample = -5; sample < nbMeasures; sample++) { // 5 Warmup runs and samples to average tests (in python, later)
+            for (int sample = -10; sample < nbMeasures; sample++) { // 5 Warmup runs and samples to average tests (in python, later)
```

Comment:
You should consider using a dedicated `nbWarmups` variable.

Also, update the comment to reflect your change in the number of warmup runs:
```suggestion
        for (int sample = -10; sample < nbMeasures; sample++) { // 10 Warmup runs and samples to average tests (in python, later)
```